opam

```diff
@@ -34,9 +34,11 @@ remove: [
 depends: [
   "ocamlfind" {build}
   "re"
-  "sexplib" {>="109.53.00"}
+  "sexplib" {>= "113.24.00"}
```

Comment:
Is your question about the constraint on the version or the existence of the dependency? The library links `sexplib` findlib and uses its code at run-time. The constraint and the dependency aren't strictly necessary as `ppx_sexp_conv` implies both. I guess I don't see an issue with how this is now but I think I would be fine with dropping the constraint. I would have a harder time dropping the dependency and letting it be implied because then opam knows less about which libraries are actually in use.